src/js/components/sharedComponents/ReadMoreUpdated.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * ReadMoreUpdated.jsx
+ * Created by Nick Torres 06/15/2022
+ */
+
```

Comment:
just wondering why we needed a new component here instead of updating the existing one and wondering if we could name the component so it's clear how this one is different than the old one

Reply:
probably because i have PTSD from trying to use any shared components and it has come back to bite me in some way or another, I'll try to update the previous ReadMore component and see if it still looks okay/doesn't break anything that uses the current ReadMore

Reply:
I updated the old ReadMore component, there are cells on the data dictionary page that can be compared with prod to be sure i didn't break anything
![image](https://user-images.githubusercontent.com/102241022/175995821-2f14cf75-2820-4ebf-8e14-f5177544eada.png)